storage/innobase/xtrabackup/doc/source/installation/docker.rst

```diff
@@ -67,6 +67,10 @@ intend to use with *Percona XtraBackup*.
 As soon as Percona Server for MySQL runs, add some data to it. Now, you are
 ready to make backups with Percona XtraBackup.
 
+.. important::
```

Comment:
Highlight this as Bold

Reply:
The directive cannot be made bold, but I've made the text in the note bold.